boolean.go

```diff
@@ -10,9 +10,9 @@ package python3
 /*
 #include "Python.h"
 #include "macro.h"
-#include "type.h"
 */
 import "C"
+import "unsafe"
```

Comment:
Could we use the same syntax than for other import (ie one import line with parenthesis) ?

Reply:
Yes, sure

Reply:
Though we have to keep the `import "C"` line for cgo